### PR TITLE
Add theFLARE.net

### PR DESCRIPTION
**Pull request by trueflare**
## Type of change

- [ X ] Add new directory for a server
- [ X ] Update directory of a server
- [ ] Remove directory of a server
- [ X ] Documentation Update

## Checklist

- [ X ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ X ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ X ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/theflare/manifest.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "gamemodes": {
+    "lifesteal": {
+      "name": "Lifesteal SMP",
+      "color": "#cf3e3e",
+      "versions": "1.20.1+"
+    }
+  },
+  "discord": {
+    "server_id": 1119215380745945098,
+    "rename_to_minecraft_name": true
```
Comment (maaaathis) [inline minecraft_servers/theflare/manifest.json:22]: `rename_to_minecraft_name` = `true` is only allowed for LabyMod partners. See here: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md?plain=1#L27

File: minecraft_servers/theflare/manifest.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "server_name": "theflare",
+  "nice_name": "theFLARE",
+  "direct_ip": "trueflare.net",
+  "supported_languages": [
+    "en"
+  ],
+  "social": {
+    "web": "https://trueflare.net",
```
Comment (maaaathis) [inline minecraft_servers/theflare/manifest.json:9]: Is your website down? 
`Verbindung fehlgeschlagen: Access denied for user 'JamesBond'@'localhost' (using password: YES)`

**Comment by maaaathis** [comment]
Is your server already online?
image

**Review by maaaathis — COMMENTED** [review]
Is your server already online?
image

**Comment by DoJapHD** [comment]
Server offline/maintenance; reopen PR if server is released and available

**Pull request closed without merging**